src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -182,10 +208,16 @@ object VEP {
 
     info(s"vep: annotated ${ annotations.count() } variants")
 
+    val (globalValue, globalType) =
+      if (csq)
+        (Row(csqHeader.getOrElse("")), TStruct("vep_csq_header" -> TString()))
```

Comment:
The path is hardcoded as `vep_csq_header` but in the documentation, you have `'name'_csq_header`.

Reply:
It's not hardcoded with respect to the returned table or matrix table

Reply:
ok, got it